client/packages/programs/src/schema_types.ts

```diff
@@ -1,7 +1,21 @@
+import { EncounterNodeStatus } from '@common/types';
+import { Clinician } from 'packages/system/src/Clinician';
```

Comment:
causes issues with tests (yes, I've explored using eslint to pick this up.. no success yet)

I've added this to `packages/system/src/index.ts`
```
export * from './Clinician';
```

and updated the import to 
```suggestion
import { Clinician } from '@openmsupply-client/system';
```

Reply:
Sorry have to remember to check these myself!